store/data.js

```diff
@@ -212,7 +212,9 @@ export const actions = {
   ) {
     try {
       commit('setTransactionsLoading', true)
-      const transactions = await api.getTransactions(address, pageNumber)
+      // Deactivate for now as not working on Stargate
+      // const transactions = await api.getTransactions(address, pageNumber)
+      const transactions = await []
```

Comment:
```suggestion
      const transactions = await api.getTransactions(address, pageNumber)
```

let's not check that in

Reply:
Ping